components/Favoring/FavoringFunctions.js

```diff
@@ -0,0 +1,11 @@
+export function checkFavorites(favorites, number) {
+  return favorites.includes(number);
+}
+
+export function handleFavorites(favorites, number) {
```

Comment:
I recommend the name toggleFavorites, since "handle" is very generic and gives no information about what the function actually does. (In fact it sounds like a handle-function that does something more intruding like state setting, persistence, etc.)

Reply:
Thanks, changed it